hyp3_autorift/process.py

```diff
@@ -201,10 +201,12 @@ def get_s1_primary_polarization(granule_name):
 
 
 def create_fft_filepath(path: str):
-    name, extension = Path(path).name.split('.')
-    out_name = name + '_fft.' + extension
-    out_path = str(Path('.').resolve() / out_name)
-    return out_path
+    parent = (Path.cwd() / 'fft').resolve()
+    if not parent.exists():
+        parent.mkdir()
```

Comment:
```suggestion
    parent.mkdir(exist_ok=True)
```
https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir